watson_developer_cloud/tone_analyzer_v3.py

```diff
@@ -13,15 +13,25 @@
 # limitations under the License.
 
 """
-The v3 Tone Analyzer service
-(https://www.ibm.com/watson/developercloud/tone-analyzer.html)
+The IBM Watson™ Tone Analyzer Service uses linguistic analysis to detect
```

Comment:
I think the `™` is not an ASCII character. You need to add `# coding: utf-8` to the top of the file. It will be great if you can add it to the generator so that we don't run into this issue in the future.